mkt/settings.py

```diff
@@ -1001,8 +1001,13 @@ def JINJA_CONFIG():
 READ_ONLY = False
 
 REDIS_BACKENDS = {
```

Comment:
The nice thing about the single string parse is you can easily override this with an environment variable, which is how it is currently on master for docker. It's something I broke here and need to fix.